pages/implementation/devs-101.mdx

```diff
@@ -0,0 +1,286 @@
+---
+layout: 'guide'
+title: 'Avo 101 for developers'
+# abstract: 'Developer's overview of Avo'
+---
+
+import Link from '../../components/Link';
+
+The use case of Avo for developers is twofold.
+
+- Avo generated file, which is the code representation of your tracking plan, containing Avo Functions
```

Comment:
I'm wondering if we can use this opportunity to frame the value props?
```suggestion
- Implementing reliable tracking fast with the type-safe Avo Functions. Avo Functions are type-safe, code generated representation of your tracking plan.
```

Reply:
How about:

- Avo generated file, which is the type-safe code representation of your tracking plan, containing Avo Functions, allows you to implementing reliable tracking fast.

- Avo Inspector SDKs, allow you to plug in your existing tracking to Avo to analyze it and help you migrate to type-safe implementation.

Reply:
Shipit!